applications/GeoMechanicsApplication/.git-blame-ignore-revs

```diff
@@ -1,4 +1,5 @@
 # Clang-format commits
 42812c76886157f305718ef13f66c6b4274f3780
 ab13f86f8770ccaaa23ef69c63f14f443da8bea0
-206d114e8082ce70ebf375e2a4d4e84f366c6696
\ No newline at end of file
+206d114e8082ce70ebf375e2a4d4e84f366c6696
+f051af1a6a180c67b27c0deb29062999629ededc
```

Comment:
This is such that the previous reformatting commit is not shown when doing 'git blame'